src/emm/emm_cli.py

```diff
@@ -274,5 +287,32 @@ def git_commit(ctx: click.Context, commit_message: str) -> None:
     orchestrator.git_commit_modules(commit_message)
 
 
+@cli.command(
+    context_settings=dict(max_content_width=100, ignore_unknown_options=True, allow_extra_args=True)
+)
+@click.pass_context
+def pull_request(ctx: click.Context) -> None:
+    """
+    Create pull request for the changes in each module.
+
+    Before use this command, you have to authenticate to github by 'gh auth login'.
+
+    Any enabled modules with committed changes would create a pull request,
+    each pull request would contain links of other modules' pull request as comment.
+
+    Use --title and --body to create a pull request or use --fill autofill these values from git commits.:
```

Comment:
We should probably have a check that either `--title` and `--body` are provided as arguments or `--fill` are provided.

Reply:
After our discussion, I think I would let github cli to handle those cases. I add a comment that we would let github cli error out and prompt error messages when required options are missing. 

If we are handling those option on the click option level or orchestrator.git_pull_request level, it would require several combinations like "-t"&"-b" or "--title"&"-b" or  "--title" and "--body" or "-t"&"--body" or "-f" or "--fill". I think that make our coding style not consistent and looks cumbersome. 

But if we want handle those options, I can still add those checks back, I just feel it may not elegant.